examples/python/interface.py

```diff
@@ -136,14 +136,26 @@ def control_test(control_module='', start_time=0, warmup_period=0, length=24*360
     # Initialize test case
     print('Initializing test case simulation.')
     if scenario is not None:
-        # Initialize test with a scenario time period
+        # Initialize test with a scenario time period specified
         res = check_response(requests.put('{0}/scenario'.format(url), json=scenario))['time_period']
         print(res)
-        # Record test simulation start time
-        start_time = int(res['time'])
-        # Set final time and total time steps to be very large since scenario defines length
-        final_time = np.inf
-        total_time_steps = int((365 * 24 * 3600)/step)
+        
+        # Initialize test with a scenario without time period specified
+        if res == None:
```

Comment:
@terrancelu92 Can you remind me, why did you add this if-else, since you can specify the ``scenario`` parameter as ``None`` and go to line 160, which initializes the simulation according to start_time and warm_up period instead of with a scenario.  This if-else between lines 144-150 seems to duplicate the functionality.

Reply:
@dhblum, I vaguely recall there being a reason for not setting the `scenario `parameter to `None` and going to line 160. Let me look into it and get back to you. Thanks for bringing this to my attention.

Reply:
@terrancelu92 Ah ok, I see the issue, and think your approach is ok.  I tried thinking a bit if there was a more streamlined approach, but couldn't come up with anything particularly better.  I think I will just adjust the comments around the code though to make it more clear to a reader what the if-else's are for.